Resources/Prototypes/GameRules/roundstart.yml

```
@@ -114,7 +114,8 @@
   id: Nukeops
   components:
   - type: GameRule
-    minPlayers: 20
+#    minPlayers: 20 # Commented out on Harmony
+    minPlayers: 35 # Updated minimum required players for Harmony
```

You don't have to keep the old line when you're just changing one value
```suggestion
    minPlayers: 35 # harmony: Increased minimum player count for nukies
```